pkg/scheduler/framework/plugins/migration/utils.go

```diff
@@ -124,3 +125,26 @@ func PredicateMetadata(state *framework.CycleState) interface{} {
 	}
 	return meta
 }
+
+// CovertStateRefToPredMeta checks if 'stateRef' is nil, if it is, return nil;
+// otherwise covert it to predicates metadata and return.
+func CovertStateRefToPredMeta(stateRef interface{}) (predicates.Metadata, bool) {
+	if stateRef == nil {
+		return nil, true
+	}
+	meta, ok := stateRef.(predicates.Metadata)
+	return meta, ok
+}
+
+// RegisterFilterPluginFunc is a function signature used in method RegisterFilterPlugin()
+// to register a Filter Plugin to a given registry.
+type RegisterFilterPluginFunc func(reg *framework.Registry, plugins *schedulerapi.Plugins, pluginConfigs []schedulerapi.PluginConfig)
+
+// RegisterFilterPlugin returns a function to register a Filter Plugin to a given registry.
+func RegisterFilterPlugin(pluginName string, pluginNewFunc framework.PluginFactory) RegisterFilterPluginFunc {
```

Comment:
I don't believe those two functions this will be used anywhere other than unit tests, and they are not quite related to migration because they will live beyond migration, right? if yes, how about we put those in ```scheduler/testing/framework_helpers.go```?

Reply:
Correct. Sounds fair to move to `testing` package.